packages/kbn-plugin-generator/sao_template/template/package_template.json

```diff
@@ -1,41 +0,0 @@
-{
```

Comment:
Why did we decide to remove this? I think it's still useful for plugins to be able to have their own dependencies and we should set them up to do that. The yarn workspaces config we have should support installing dependencies when running `yarn kbn bootstrap` for plugins in this directory.

In addition, I think having them use our eslint config by default is a good idea since it helps prevent some common problems. However, they're not currently depending on it, this just relies on the plugin being in a specific directory. I suspect we won't need to require this once #53976 moves further along.

I also expect that they'll need to depend on `@kbn/config-schema` in order to do anything useful on the server-side.

Reply:
@joshdover I removed it because I was able to create and run plugins without it. I prefer to have a minimal structure, then re-introduce things based on need.

What do you think?

Reply:
I believe the primary audience for this package is 3rd parties. With the current system, there's a bit of complexity involved in setting up a package.json file to work with Kibana plugins (eg. the `link:` dependencies). It seems to me we should help them do this, at least as an option, since I imagine most plugins will want to use some dependencies of their own (though that is an assumption).

Reply:
We can do this in a separate follow-up PR.

Reply:
Opened an issue for addressing translations and packages.json

https://github.com/elastic/kibana/issues/56652